console/src/test/scala/io/joern/console/cpgqlserver/CPGQLServerTests.scala

```diff
@@ -40,91 +40,62 @@ class CPGQLServerTests extends AnyWordSpec with Matchers {
     ujson.read(getResponse.bytes)
   }
 
-  "CPGQLServer" should {
-
-    "allow websocket connections to the `/connect` endpoint" in Fixture() { host =>
-      val wsMsgPromise = scala.concurrent.Promise[String]()
-      cask.util.WsClient.connect(s"$host/connect") { case cask.Ws.Text(msg) =>
-        wsMsgPromise.success(msg)
-      }
-      val wsMsg = Await.result(wsMsgPromise.future, DefaultPromiseAwaitTimeout)
-      wsMsg shouldBe "connected"
-    }
-
-    "allow posting a simple query without any websocket connections established" in Fixture() { host =>
-      val postQueryResponse = postQuery(host, "1")
-      postQueryResponse.obj.keySet should contain("success")
-      val UUIDResponse = postQueryResponse("uuid").str
-      UUIDResponse should not be empty
-      postQueryResponse("success").bool shouldBe true
-    }
-
-    "disallow posting a query when request headers do not include a valid authentication value" in Fixture() { host =>
-      assertThrows[RequestFailedException] {
-        postQuery(host, "1", authHeaderVal = "Basic b4df00d")
+  /** These tests happen to fail on github actions for the windows runner with the following output: WARNING: Unable to
+    * create a system terminal, creating a dumb terminal (enable debug logging for more information) Apr 21, 2022
+    * 3:08:54 PM org.jboss.threads.Version <clinit> INFO: JBoss Threads version 3.1.0.Final Apr 21, 2022 3:08:55 PM
+    * io.undertow.server.HttpServerExchange endExchange ERROR: UT005090: Unexpected failure
+    * java.lang.NoClassDefFoundError: Could not initialize class org.xnio.channels.Channels at
+    * io.undertow.io.UndertowOutputStream.close(UndertowOutputStream.java:348)
+    *
+    * This happens for both windows 2019 and 2022, and isn't reproducable elsewhere. Explicitly adding a dependency on
+    * `org.jboss.xnio/xnio-api` didn't help, as well as other debug attempts. So we gave up and disabled this
+    * specifically for github actions' windows runner.
+    */
+  val isGithubActions = System.getenv("GITHUB_ACTIONS") == "TRUE"
+  val isWindows       = System.getProperties.getProperty("os.name", "<unknown>").toLowerCase.contains("windows")
```

Comment:
You may use `scala.util.Properties.isWin` here.

Reply:
nice, didn't know this one, thanks for sharing